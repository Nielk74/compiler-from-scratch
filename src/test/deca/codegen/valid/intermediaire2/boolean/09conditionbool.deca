{
int x = 5;
int y = 10;
boolean t = true || false;
boolean f = false && true;
if(((x < y) == t ) && ((false != t ) || !(true != (x == y)))){
    println("((x < y) == t ) && ((false != t ) || !(true != (x == y)))");
}else{
    println("!(((x < y) == t ) && ((false != t ) || !(true != (x == y))))");
}
}
