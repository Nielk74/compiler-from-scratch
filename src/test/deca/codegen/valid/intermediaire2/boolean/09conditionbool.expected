((x < y) == true ) && ((false != true ) || !(true != (x == y)))
